docarray/helper.py

```diff
@@ -455,3 +459,37 @@ def _safe_cast_int(value: Union[str, int, float]) -> int:
     if isinstance(value, float) and not value.is_integer():
         raise ValueError(f"Can't safely cast {value} to an int")
     return int(value)
+
+
+def _get_array_info(da: 'DocumentArray'):
+    all_attrs = da._get_attributes('non_empty_fields')
+    # remove underscore attribute
+    all_attrs = [tuple(vv for vv in v if not vv.startswith('_')) for v in all_attrs]
+    attr_counter = Counter(all_attrs)
+
+    all_attrs_names = set(v for k in all_attrs for v in k)
+    _nested_in = []
+    if 'chunks' in all_attrs_names:
+        _nested_in.append('chunks')
+
+    if 'matches' in all_attrs_names:
+        _nested_in.append('matches')
+
+    is_homo = len(attr_counter) == 1
+
+    _nested_items = []
+    if not is_homo:
+        for _a, _n in attr_counter.most_common():
+            if _n == 1:
+                _doc_text = f'{_n} Document has'
+            else:
+                _doc_text = f'{_n} Documents have'
+            if len(_a) == 1:
+                _text = f'{_doc_text} one attribute'
+            elif len(_a) == 0:
```

Comment:
can't this be used in the `raw_summary`?

Reply:
why not it's already used ?

Reply:
btw I just applied refactoring I realized I wasn't using helper function